Logging model files to ./chkp/12-31-2022_21:14:43
TRAINING MODEL: regressor
Training on CPU
************* EPOCH: 1 *************
<class 'builtin_function_or_method'>
Pointnet.py
  0%|                                                                                                                                        | 0/1 [00:00<?, ?it/s][W NNPACK.cpp:79] Could not initialize NNPACK! Reason: Unsupported hardware.
  0%|                                                                                                                                        | 0/1 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "train_frustum_neonet.py", line 101, in <module>
    main(args)
  File "train_frustum_neonet.py", line 89, in main
    trainer.fit()
  File "/Users/stephennelson/Projects/Learning-Frustum-Proposals/trainers/base_trainer.py", line 346, in fit
    y_hat = self._batch_prediction(inputs.double()).to(self.device)
  File "/Users/stephennelson/Projects/Learning-Frustum-Proposals/trainers/learned_frustum_trainer.py", line 54, in _batch_prediction
    y_hat = self.model(
  File "/opt/anaconda3/envs/thesis/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/Users/stephennelson/Projects/Learning-Frustum-Proposals/models/frustum_neonet.py", line 30, in forward
    h = self.encoder(points)
  File "/opt/anaconda3/envs/thesis/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/Users/stephennelson/Projects/Learning-Frustum-Proposals/models/pointnet.py", line 161, in forward
    x = self.conv_layers[0](x)
  File "/opt/anaconda3/envs/thesis/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/opt/anaconda3/envs/thesis/lib/python3.8/site-packages/torch/nn/modules/conv.py", line 298, in forward
    return self._conv_forward(input, self.weight, self.bias)
  File "/opt/anaconda3/envs/thesis/lib/python3.8/site-packages/torch/nn/modules/conv.py", line 294, in _conv_forward
    return F.conv1d(input, weight, bias, self.stride,
RuntimeError: expected scalar type Double but found Float